functions/New-DbaLogin.ps1

```diff
@@ -493,12 +508,31 @@ function New-DbaLogin {
                                 }
                             }
                         }
+
+                        #Process the MustChangePassword property
+                        if ($currentPasswordMustChange -ne $newLogin.MustChangePassword) {
+                            try {
+                                $newLogin.ChangePassword($SecurePassword, $true, $true)
```

Comment:
If it requires password change, then we need to validate that the password is specified when the MustChangePassword 
 parameter is used.

Reply:
Added validation on the top of the function.
Thanks!

Reply:
Looks good!